hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/regression/system/DabEnabledUpgradeTest.java

```diff
@@ -88,7 +87,7 @@
  * the config version is still zero.
  */
 @Tag(UPGRADE)
-@Order(MAX_VALUE)
+@Order(Integer.MAX_VALUE - 2)
```

Comment:
This strange expression `Integer.MAX_VALUE - 2` appears in no other `@Order` so isn't obvious.  Why not leave a comment here explaining what it says in the underlying issue?  And is it `- 2` instead of `- 1` so that it preceeds `StreamValidationTest` in addition to `LogValidationTest`?

Reply:
Ah yes I should add this detail @david-bakin-sl , will do in next PR. 🙏